src/main/java/com/templateproject/api/entity/Comment.java

```diff
@@ -0,0 +1,56 @@
+package com.templateproject.api.entity;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+
+@Entity
+public class Comment {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String text;
+    private String author;
```

Comment:
Is the String author temporary or do you plan to keep it later (if temporary can you put a comment on top to say that it will be remove later when the User will be implement). If you plan to keep it how do you plan to use it?

Reply:
All the class User is destined to be deleted later with spring security/JWT. Eventually, it will become an entity and we'll use the author attribute to identify the user who post the comments. I can change the attribute with username instead.

Reply:
Nevermind, I misread.

Reply:
I made a mistake, comment doesn't need an author cause we'll have a User later.